pkg/config/app/kuma-cp/default_config_test.go

```diff
@@ -37,11 +37,14 @@ var _ = Describe("Default config", func() {
 			},
 		}
 
+		defaultCfg := DefaultConfig()
+		defaultCfg.GuiServer.Directory = "/path-to-kuma-cp/gui" // default retrieves path to kuma-cp which is not stable in tests
```

Comment:
Let's remove environment-specific configuration from `DefaultConfig()` and rollback these changes (see comments to `DefaultGuiServerConfig()`)